enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverter.java

```diff
@@ -142,11 +142,12 @@ private FormData prepareFormData(
         return formData;
     }
 
-    private static StringSubstitutor createStringSubstitutor(final Map<String, String> parameters) {
+    static StringSubstitutor createStringSubstitutor(final Map<String, String> parameters) {
```

Comment:
May we test it without making it _package private_? According to the test coverage, current tests go through both branches.